Content of my Portfolio

Josphat205 opened the pull request:
this is what I added

- Added Html file
- Added CSS file 
- Added Linters files
- Added Stylelintrc files
- Added a configuration with linters commands
- Added a Readme file
- Added gitignore file

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,82 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!---top navigation-->
+    <nav class="my-topnav">
+      <h2 class="logo"><a href="#">Jloman</a></h2>
+      <span><ion-icon name="menu-outline" class="menu"></ion-icon></span>
+      <ul>
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+          <li><a href="#message"><ion-icon name="mail-outline"></ion-icon></a></li>
+      </ul>
+    </nav>
```
- [ ] Kindly wrap the section with the `header` tag for the best semantic HTML.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,82 @@
+                <h5>Work</h5>
+                <span><ion-icon name="arrow-down-outline"></ion-icon></span>
+            </div>
+            <div class="social-links">
+                <span><a href="#"><ion-icon name="logo-github"></ion-icon></a></span>
+                <span><a href="#"><ion-icon name="logo-linkedin"></ion-icon></a></span>
+                <span><a href="#"><ion-icon name="logo-twitter"></ion-icon></a></span>
+                <span><a href="#"><ion-icon name="logo-facebook"></ion-icon></a></span>
+            </div>
```
- [ ] The list of icons should use a ul tag to group them, and li per icon; each icon has a link (a tag). So kindly implement it as it is a list.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,82 @@
+                look through part of my work and experince! if you like what you see and have a project you need coded, dont hesitate contacting me.
+            </p>
+            <hr>
+            <div class="Knowledge">
+              <div class="language">
+                 <h2>Language</h2>
+                 <h5>Javascript</h5>
+                 <h5>Ruby</h5>
+                 <h5>HTML</h5>
+                <h5>CSS</h5>
+              </div>
+             <div class="frameworks">
+                 <h2>Frameworks</h2>
+                <h5>Bootstrap</h5>
+                <h5>Ruby on Rails</h5>
+                <h5>PSPec</h5>
+                <h5>CapyBara</h5>
+               <h5>Selenium</h5>
+             </div>
+             <div class="skills">
+                 <h2>Skills</h2>
+                <h5>Github</h5>
+                <h5>Gitlab</h5>
+                <h5>Codepen</h5>
+                <h5>Terminal</h5>
+               <h5>Codekit</h5>
+             </div>
+            </div>
```
- [ ] I would suggest you stick as much as possible to Figma-design
- [ ] Kindly implement the same background as the Figma design.
- [ ] If you see the language and framework part then it should be aligned as the same line, but in your project, it is on a different line. Kindly fix it.
- [ ] I would suggest you make equal spacing in your social icons and remove the works and put it in after the icons.
- Your design
![button design](https://user-images.githubusercontent.com/81727988/164388607-0bdc5cc0-6b9c-4592-a9e0-35c89fa0778
![button design](https://user-images.githubusercontent.com/81727988/164388767-dde228e2-e44e-4031-bc4a-800218f6f785.png)
 - Figma design
 
![Figmadesign](https://user-images.githubusercontent.com/81727988/164388863-eed98636-3733-4ad9-b51c-3c107f35c44e.png)

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,82 @@
+        <img src="img/pic.png" alt="" class="img1">
+        <img src="img/pic.png" alt="" class="img2">
+        <div class="frame">
+            <h1>Josphat Kiploman</h1>
+            <h3>Software Developer</h3>
+        </div>
+        <div class="support-text">
+            <p>I can help you build a product, feature or website
+                look through part of my work and experince! if you like what you see and have a project you need coded, dont hesitate contacting me.
```
- [ ] It would be better if you make the spacing equally from left and right. See the Figma design
- [ ] Kindly change the color, and font of the `h1` tag it is totally different from Figma's design, and change the background as well.
- your design
![upper part](https://user-images.githubusercontent.com/81727988/164389990-ce213cec-0da8-4a87-98c6-f2c79a92f130.png)
- Figma design
![Figmadesign](https://user-images.githubusercontent.com/81727988/164390033-6d1ca07d-92e3-459f-9261-c13b388ca9bf.png)

roshan-bajgain reviewed (changes requested):
Hi @Josphat205 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [ ] Kindly update the README file. use this [Template](https://github.com/microverseinc/readme-template/blob/master/README.md) for making more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Josphat205 commented:
what I changed in my portfolio

- Added Figma background
- Replaced Span with ul list
- Used H1 to match Figma color
- Arranged framework, Language, and skills to match the project requirement
- Updated my Readme file

vic778 commented on index.html:
```diff
@@ -0,0 +1,80 @@
+        <img src="img/pic.png" alt="" class="img1">
+        <img src="img/pic.png" alt="" class="img2">
+        <div class="frame">
+            <h1>Josphat Kiploman</h1>
+            <h3>Software Developer</h3>
```
Kindly the `h1` should have a class because it has a particular color. and fix all changes request by the previous review. remember that You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). as is require for this project.

vic778 reviewed (changes requested):
## Hello @Josphat205   

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://user-images.githubusercontent.com/65068771/141811112-252360b9-757f-4279-bb3b-b7a2a4675ad3.gif)
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AnselemOdims reviewed (approved):
### Hi  @Josphat205 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions
- [ ] At the moment there is horizontal scroll on the page. Please endeavour to fix them because it does not give a good user experience

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
